modules/saml/lib/Message.php

```diff
@@ -213,7 +213,10 @@ public static function validateMessage(
             }
         }
 
-        if ($enabled === null) {
+        // If not specifically set to false, the signature must be checked to conform to SAML2INT
+        if (isset($_REQUEST['Signature']) && ($enabled !== false)) {
```

Comment:
This does not work with the POST binding, does it?

Reply:
Good catch! Fixed it